bp_common/src/include/bp_common_aviary_pkg.vh

```diff
@@ -424,7 +432,424 @@ package bp_common_aviary_pkg;
       ,cc_y_dim  : 4
       ,ic_y_dim  : 1
       ,mc_y_dim  : 0
-      ,ac_x_dim  : 0
+      ,cac_x_dim : 0
+      ,sac_x_dim : 0
+
+      ,vaddr_width: 39
+      ,paddr_width: 40
+      ,asid_width : 1
+
+      ,branch_metadata_fwd_width: 28
+      ,btb_tag_width            : 10
+      ,btb_idx_width            : 6
+      ,bht_idx_width            : 9
+      ,ras_idx_width            : 2
+
+      ,itlb_els             : 8
+      ,dtlb_els             : 8
+
+      ,lce_sets             : 64
+      ,lce_assoc            : 8
+      ,cce_block_width      : 512
+      ,cce_pc_width         : 8
+
+      ,l2_sets : 256
+      ,l2_assoc: 4
+
+      ,fe_queue_fifo_els: 8
+      ,fe_cmd_fifo_els  : 4
+
+      ,async_coh_clk       : 0
+      ,coh_noc_max_credits : 8
+      ,coh_noc_flit_width  : 128
+      ,coh_noc_cid_width   : 2
+      ,coh_noc_len_width   : 3
+
+      ,async_mem_clk         : 1
+      ,mem_noc_max_credits   : 8
+      ,mem_noc_flit_width    : 64
+      ,mem_noc_cid_width     : 2
+      ,mem_noc_len_width     : 4
+
+      ,async_io_clk         : 1
+      ,io_noc_did_width     : 3
+      ,io_noc_max_credits   : 16
+      ,io_noc_flit_width    : 64
+      ,io_noc_cid_width     : 1
+      ,io_noc_len_width     : 4
+      };
+
+  localparam bp_proc_param_s bp_accelerator_single_core_cfg_p = 
```

Comment:
Have all these been tested? Since it's just a demo, I would suggest maybe only add 1 single core and 1 multicore config

Reply:
Another thing is that we probably want to move a model with separate core parameters and soc parameters (#239), to avoid this cross-product issue

Reply:
Not yet, I've just tested single, dual, and quad configs with accelerator. Agree, single and quad should be enough for demo purpose.

Reply:
Quad core works right? 

Yeah, delete the other configs for now and we'll revisit it during #239

Reply:
yes, it works